build/media_source/templates/site/cassiopeia/scss/blocks/_tags.scss

```
@@ -2,7 +2,7 @@
 
 .tags {
   .list-inline-item {
-    margin-bottom: .5rem;
+    margin: .25rem .25rem .5rem .25rem;
```

```suggestion
    margin: .25rem .25rem .5rem;
```
That’s what the linter complains about in drone. It says unexpected long hand value where short hand value is expected.